### ISSUE TITLE
Predicted warping array has only '0's in the numpy array

### ISSUE THREAD

**Issue by Kanyq** [voxelmorph/voxelmorph#278]
Hello,

I trained my model on 3D liver scan with dimension (288, 288, 96) for 50 epochs. Then to test the predictions, I ran the "vxm_model.predict()" on my validation data of list length 2.
Structure of the list passed in predict(): [(1, 288, 288, 96, 1), (1, 288, 288, 96, 1)]

Then as the tutorial suggests I saved the outputs in the following fashion:
moved_pred = val_pred[0].squeeze()
pred_warp = val_pred[1]

For "moved_pred" I was able to generate images from the predicted numpy array but for **pred_warp** I'm getting the numpy array with **only 0s** in them.
simply put : pred_warp = val_pred[1] = 0

Any help is appreciated. Thanks

**Comment by Kanyq** [comment]
Yes I did. Thank you for checking on it though
